fulp_modules/jobs/antag_protection_overwrites.dm

```diff
@@ -1,62 +1,170 @@
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/game_mode/cult/New()
-	protected_jobs += list("Brig Physician", "Deputy")
+	protected_jobs += list(
+	"Brig Physician",
+	"Deputy",
+	)
 	. = ..()
 
 /datum/game_mode/revolution/New()
-	protected_jobs += list("Brig Physician", "Deputy")
+	protected_jobs += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/game_mode/changeling/New()
-	protected_jobs += list("Brig Physician", "Deputy")
+	protected_jobs += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/game_mode/eldritch_cult/New()
-	protected_jobs += list("Brig Physician", "Deputy")
+	protected_jobs += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /// Roundstart dymamic rulesets.
 /datum/dynamic_ruleset/roundstart/traitor/New()
-	protected_roles += list("Brig Physician", "Deputy")
+	protected_roles += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/dynamic_ruleset/roundstart/traitorbro/New()
-	protected_roles += list("Brig Physician", "Deputy")
+	protected_roles += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/dynamic_ruleset/roundstart/changeling/New()
-	protected_roles += list("Brig Physician", "Deputy")
+	protected_roles += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/dynamic_ruleset/roundstart/heretics/New()
-	protected_roles += list("Brig Physician", "Deputy")
+	protected_roles += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/dynamic_ruleset/roundstart/bloodcult/New()
-	protected_roles += list("Brig Physician", "Deputy")
+	protected_roles += list(
+	"Brig Physician",
+	"Deputy",
+	)
 	. = ..()
 
 /datum/dynamic_ruleset/roundstart/revs/New()
-	protected_roles += list("Brig Physician", "Deputy")
+	protected_roles += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
+/// Haha, families.
 /datum/dynamic_ruleset/roundstart/families/New()
-	protected_roles += list("Brig Physician", "Deputy")
+	protected_roles += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /// Dynamic Latejoin rulesets
 /datum/dynamic_ruleset/latejoin/infiltrator/New()
```

Comment:
Correct me if I am wrong, but Infiltrators game mode literally gives 0 fucks about your chosen job and ignores it completely.
It's here only to prevent infils spawning with mindshields(which is, I believe, a bug).